include/znc/Client.h

```diff
@@ -111,12 +113,24 @@ class CClient : public CIRCSocket {
 			  m_sNetwork(""),
 			  m_sIdentifier(""),
 			  m_spAuth(),
-			  m_ssAcceptedCaps()
+			  m_ssAcceptedCaps(),
+			  m_mCoreCaps({{"multi-prefix", std::make_tuple(false, [this](bool bVal) { m_bNamesx = bVal; })},
+			               {"userhost-in-names", std::make_tuple(false, [this](bool bVal) { m_bUHNames = bVal; })},
+			               {"echo-message", std::make_tuple(false, [this](bool bVal) { m_bEchoMessage = bVal; })},
+			               {"server-time", std::make_tuple(false, [this](bool bVal) { m_bServerTime = bVal; })},
+			               {"batch", std::make_tuple(false, [this](bool bVal) { m_bBatch = bVal; })},
+			               {"cap-notify", std::make_tuple(false, [this](bool bVal) { m_bCapNotify = bVal; })},
+			               {"away-notify", std::make_tuple(true, [this](bool bVal) { m_bAwayNotify = bVal; })}})
```

Comment:
Ah, it's fine. Probably std::pair instead of std::tuple would allow `{}`, but whatever...